compiler/src/test/scala/dx/compiler/CompilerTest.scala

```diff
@@ -1119,8 +1119,32 @@ class CompilerTest extends AnyFlatSpec with Matchers with BeforeAndAfterAll {
     ignoreReuseFlag shouldBe Some(JsBoolean(true))
   }
 
+  it should "override job-reuse flag on applet" taggedAs NativeTest in {
```

Comment:
add test on ApplicationCompiler to cover the issue: the `ignoreReuse=false` from extras should override the task runtime setting.
https://github.com/dnanexus/dxCompiler/blob/85c60e4ede211441fcb8c93c7103f1dd6f190d92/compiler/src/test/resources/compiler/add_runtime_hints.wdl#L23